test/setchplenv/verify_setchplenv_scripts.py

```diff
@@ -134,21 +134,24 @@ def get_path_var(var_name):
         path_parts = get_path_var('PATH')
         self.assertTrue(len(path_parts) >= 2)
 
-        path_chpl = path_parts[0].replace(self.chpl_home, 'CHPL_HOME').replace(
-            self.chpl_platform, 'CHPL_PLATFORM')
-        path_util = path_parts[1].replace(self.chpl_home, 'CHPL_HOME')
+        expected_path_chpl = os.path.join(self.chpl_home, 'bin', self.chpl_platform)
+        expected_path_util = os.path.join(self.chpl_home, 'util')
 
-        self.assertEqual('CHPL_HOME/bin/CHPL_PLATFORM', path_chpl)
-        self.assertEqual('CHPL_HOME/util', path_util)
+        actual_path_chpl = path_parts[0]
+        actual_path_util = path_parts[1]
+
+        self.assertEqual(os.stat(expected_path_chpl), os.stat(actual_path_chpl))
+        self.assertEqual(os.stat(expected_path_util), os.stat(actual_path_util))
```

Comment:
I see that os.stat being equal for the two paths might work because they have the same device and inode number, but I'm suspicious that on e.g. Cygwin there won't be any inode numbers. Do you have reason to believe that this will be portable?